python/tvm/auto_scheduler/auto_schedule.py

```diff
@@ -175,17 +236,6 @@ def auto_schedule(task, search_policy='default', tuning_options=None):
         raise ValueError("Invalid task: " + task +
                          " . `auto_scheduler.auto_schedule` expects a SearchTask.")
 
-    if isinstance(search_policy, str):
-        if search_policy == 'default':
-            # TODO(jcf94): This is an example policy for minimum system, will be upgrated to
-            # formal search policy later.
-            search_policy = EmptyPolicy()
-        else:
-            raise ValueError("Invalid search policy: " + search_policy)
-    elif not isinstance(search_policy, SearchPolicy):
-        raise ValueError("Invalid search policy: " + search_policy +
-                         " . `auto_scheduler.auto_schedule` expects a SearchPolicy or a string.")
-
-    sch, tensors = _ffi_api.AutoSchedule(task, search_policy,
-                                         tuning_options if tuning_options else TuningOptions())
+    # TODO(jcf94): Remove EmptyPolicy after finish the porting of SketchSearchPolicy
+    sch, tensors = _ffi_api.AutoSchedule(search_policy or EmptyPolicy(task), tuning_options)
```

Comment:
Should we make sketch search policy as the default one? Otherwise I can imagine lots of people will ask in the discuss forum about why auto scheduler doesn't do any search...